### PR TITLE
Fish factory hse room

### PR DESCRIPTION
**Pull request by Siggiboyy**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Feature: HSE room to go between tasks and learn about Health Safety and Environment

* **What is the current behavior?** (You can also link to an open issue here)
There was no scene previously

* **What is the new behavior?** (if this is a feature change)
It is only the HSE room from an earlier project without any logic or scripts

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
Now we can add doors to go between tasks and to exit the building to go over to the other blue sector parts

### CODE REVIEW

File: Assets/AddressableAssetsData/AssetGroups/Localization-Asset-Tables-Dutch (nl).asset
Comment (MarkusJohansen) [inline Assets/AddressableAssetsData/AssetGroups/Localization-Asset-Tables-Dutch (nl).asset:1]: Should all these language file changes be made? If not please remove these from the changed files. This means all changes made in the AddressableAssetsData.

File: Assets/Boats.meta
Comment (MarkusJohansen) [inline Assets/Boats.meta:1]: Should this PR-change the boat assets as well?

File: Assets/FishFactory/Materials/Material_2006.jpg
Comment (MarkusJohansen) [inline Assets/FishFactory/Materials/Material_2006.jpg:1]: Why?😆

File: Assets/FishFactory/Materials/Old project material/Colors/Teleport-ray.jpg
Comment (MarkusJohansen) [inline Assets/FishFactory/Materials/Old project material/Colors/Teleport-ray.jpg:1]: What is this used for? Do we need it?

File: Assets/FishFactory/Materials/Old project material/Common/96956a_1000.jpg
Comment (MarkusJohansen) [inline Assets/FishFactory/Materials/Old project material/Common/96956a_1000.jpg:1]: Why is this added?

File: Assets/Textures/dark-red-texture-wallpaper-2.jpeg
Comment (MarkusJohansen) [inline Assets/Textures/dark-red-texture-wallpaper-2.jpeg:1]: Do we need jpeg files?

**Review by MarkusJohansen — APPROVED** [review]
Great work! Resolve my comments before merge. I would also consider a better naming convention for the folder within the project; "OldProject/" instead of "Old project/".🚀

File: Assets/FishFactory/Materials/Material_2006.jpg
Comment (mefriis) [reply Assets/FishFactory/Materials/Material_2006.jpg:1]: Moon?

File: Assets/Boats.meta
Comment (mefriis) [reply Assets/Boats.meta:1]: I belive this stems from the FishMerd component. Been up for discussion before? It either has to be ignored or added.

**Review by mefriis — CHANGES_REQUESTED** [review]
Check the language files mentioned by Markus.
Also, this merge contains a lot of files - but I guess most of them are related to it being an entire room/sceene that is being imported. 
No apparent errors in the unity scene. Looks good! Could do with some more lighting, and maybe the signs on the doors needs to be altered. But this is easier done at a later point when a setup between the scenes is determined.

**Review by mefriis — APPROVED** [review]

File: Assets/FishFactory/Materials/Material_2006.jpg
Comment (MarkusJohansen) [reply Assets/FishFactory/Materials/Material_2006.jpg:1]: > Moon?

Sorry, I dont understand your reply.
